### DISCUSSION TITLE
Multi-user Support (Support for multiple workspaces)

### DISCUSSION THREAD

**Discussion by oskardotglobal** [outline/outline#5216] [Ideas]
It would be good if each user could have their own workspace. I understand that there has to be a difference between the self-hosted and the hosted / enterprise version, but atleast the ability for users to have their own workspace and not having all users in one workspace by default. 

I didn't find anything about this anywhere except #4843, but if this isn't planned then maybe this should be noted somewhere.

**Comment by tommoor** [comment]
There is no plan to build multiple workspaces for the community edition

**Comment by quickstar** [comment]
@tommoor it would be nice to have an overview where the differences between the community and enterprise editions are documented?

**Reply by tommoor** [reply]
The closest right now is on the pricing page, click on "Self Managed" –

https://www.getoutline.com/pricing
